specerror/error.go

```diff
@@ -79,52 +46,14 @@ type LevelErrors struct {
 	Error *multierror.Error
 }
 
-var (
-	containerFormatRef = func(version string) (reference string, err error) {
-		return fmt.Sprintf(referenceTemplate, version, "bundle.md#container-format"), nil
-	}
-	specVersionRef = func(version string) (reference string, err error) {
-		return fmt.Sprintf(referenceTemplate, version, "config.md#specification-version"), nil
-	}
-	rootRef = func(version string) (reference string, err error) {
-		return fmt.Sprintf(referenceTemplate, version, "config.md#root"), nil
-	}
-	defaultFSRef = func(version string) (reference string, err error) {
-		return fmt.Sprintf(referenceTemplate, version, "config-linux.md#default-filesystems"), nil
-	}
-	runtimeCreateRef = func(version string) (reference string, err error) {
-		return fmt.Sprintf(referenceTemplate, version, "runtime.md#create"), nil
+var ociErrors = map[Code]errorTemplate{}
+
+func registOCIError(code Code, level rfc2119.Level, ref func(versiong string) (string, error)) {
```

Comment:
This package is just about runtime-spec requirement, so this can be `register`.  No need to have `OCI` or `error` in the name, because they're covered by the package path.

Reply:
I‘ll change it tomorrow morning.